web/src/main/java/com/futurewei/alcor/web/entity/port/PortEntity.java

```diff
@@ -443,261 +419,4 @@ public String toString() {
                     '}';
         }
     }
-
-
-    public String getVpcId() {
```

Comment:
@chenpiaoping recommended not to use @data annotation because it generates code during compilation and has Poor security and maintainability. 

Let us get back the Getter and Setter.

Reply:
fixed in 0fc306c